playground/rollup.config.js

```diff
@@ -20,6 +20,7 @@ export default (args) => {
                 preventAssignment: true,
             }),
             lwc(),
+            nodeResolve(),
```

Comment:
If the playground is broken by #5203, then other projects probably are as well. I think the fix should live there, rather than in this config. I don't know whether we'd want to add node resolution for every package, or just add a mini fix to allow LWC to be resolved.

Reply:
Makes sense. I'll push another PR.